### ISSUE TITLE
Dosent work

### ISSUE THREAD

**Issue by DomTheNoobCoder** [LEVIXDDLMAO/SONICEXE-#1]
i know why its because kade engine 1.5.4 is not fixed even tho kade dev said so i recommended you port sonic.exe fnf to 1.4.2

**Comment by LEVIXDDLMAO** [comment]
yeah this was never finshed as i keep on testing it it wouldnt work so i gave up on in it
